### PR TITLE
WaaS account federation, commit/reveal OIDC, guest accounts & PlayFab

### PR DESCRIPTION
**Pull request by patrislav**
.

### CODE REVIEW

File: packages/waas/src/auth.ts [hunk 1]
```diff
@@ -249,50 +255,78 @@ export class SequenceWaaS {
...
+    } else if ('email' in identity) {
+      return this.initiateEmailAuth(identity.email)
+    } else if ('playFabTitleId' in identity) {
+      return this.initiatePlayFabAuth(identity.playFabTitleId, identity.playFabSessionTicket)
+    }
+
+    throw new Error('invalid identity')
+  }
+
+  private async initiateGuestAuth() {
```
Comment (taylanpince) [inline packages/waas/src/auth.ts:274]: I think we should standard on `init[METHOD]Auth` for naming. It's nicer to keep it the same across the public method and internals.

File: packages/waas/src/intents/base.ts [hunk 2]
```diff
@@ -7,7 +8,7 @@ import { Session } from '../session'
 export type Intent<T> = Omit<RawIntent, 'data' | 'signatures'> & { data: T }
 export type SignedIntent<T> = Omit<RawIntent, 'data'> & { data: T }
 
-const VERSION = '1.0.0'
+const VERSION = `1 (Web ${PACKAGE_VERSION})`
```
Comment (pkieltyka) [inline packages/waas/src/intents/base.ts:11]: what is the "1" in front here for..?

Comment (patrislav) [reply packages/waas/src/intents/base.ts:11]: it's the intent version, perhaps would be better to make it explicit...

Comment (pkieltyka) [reply packages/waas/src/intents/base.ts:11]: its fine, we can leave it for now, thanks for clarifying

**Review by pkieltyka — APPROVED** [review]

**Pull request merged**